.github/SUPPORT.md

```diff
@@ -1,26 +1,150 @@
-Node.js](https://github.com/nodejs/Release#release-schedule).
+## Table of Contents
 
-When looking for support, please first search for your question in these venues:
+* [General Support](#general-support)
+  * [Where to Get Help](#where-to-get-help)
+    * [Node.js Website](#nodejs-website)
+    * [API Documentation](#api-documentation)
+    * [Node.js Help Forum](#nodejs-help-forum)
+    * [GitHub Issues](#github-issues)
+    * [StackOverflow](#stackoverflow)
+    * [Libera.chat Channel](#liberachat-channel)
+    * [Node.js Slack Community](#nodejs-slack-community)
+  * [Before Seeking Support](#before-seeking-support)
+* [Reporting a Bug](#reporting-a-bug)
+  * [How to Report and Identify](#how-to-report-and-identify)
+* [Requesting a Feature](#requesting-a-feature)
+  * [Guidelines for Feature Requests](#guidelines-for-feature-requests)
+* [Be Respectful](#be-respectful)
 
-* [Node.js Website](https://nodejs.org/en/), especially the
-  [API docs](https://nodejs.org/api/)
-* [Node.js Help](https://github.com/nodejs/help)
-* [Open or closed issues in the Node.js GitHub organization](https://github.com/issues?utf8=%E2%9C%93&q=sort%3Aupdated-desc+org%3Anodejs+is%3Aissue)
+## General Support
 
-If you didn't find an answer in the resources above, try these unofficial
-resources:
+### Where to Get Help
 
-* [Questions tagged 'node.js' on Stack Overflow](https://stackoverflow.com/questions/tagged/node.js)
-* [#node.js channel on libera.chat](https://web.libera.chat?channels=node.js&uio=d4)
-* [Node.js Slack Community](https://node-js.slack.com/)
-  * To register: [nodeslackers.com](https://www.nodeslackers.com/)
+#### Node.js Website
 
-GitHub issues are for tracking enhancements and bugs, not general support.
+The [Node.js website][] is an excellent starting point for beginners and experienced users alike to access
+documentation and resources.
 
-The open source license grants you the freedom to use Node.js. It does not
-guarantee commitments of other people's time. Please be respectful and manage
-your expectations.
+#### API Documentation
+
+For in-depth explanations and practical examples of Node.js functionalities, refer to the [API documentation][].
+
+#### Node.js Help Forum
+
+Engage with the Node.js community and seek assistance on the dedicated [Node.js Help][] forum. Here, you can
+participate in discussions, ask questions, and receive support from other developers and community members.
+
+#### GitHub Issues
+
+Interact directly with the future of Node.js by browsing or reporting issues on the [Node.js GitHub Issues][]
+page. This allows for collaboration and problem-solving within the Node.js community, allowing users to report
+bugs, suggest enhancements, and engage in discussions with the rest of the community.
+
+#### StackOverflow
+
+[StackOverflow][] is ideal for troubleshooting specific issues, seeking solutions to common problems, and learning
+from the experiences of other Node.js users and developers.
```

Comment:
Hmm.. I'm a bit reluctant to include recommendations out to sites and services that are not directly under project/foundation oversite. If we are going to add these, we should include clear indicators that the project does not endorse or maintain the content of these external services.